content/cloud/pricing-and-billing.md

```diff
@@ -0,0 +1,114 @@
+
+**How do I change my credit card used for billing by Fermyon Cloud?**
+This can be done online on the [Fermyon Cloud Dashboard](cloud.fermyon.com) under _Account Settings --> Billing_. 
+
+
+**How long is the Starter Plan available? Is there a time limit after which the free tier is unavailable to me?**
+
+At this time the Starter Plan has no time limit. As long as your resource utilization remains below the monthly limits you can use Fermyon Cloud, for free, as long as you want it to. We reserve the right to change that policy, but for now we’re thrilled to be able to provide this service to you completely free.
+
+### Invoice 
+
+**Where can I see my bill?**
+
+You can view all past invoices in the customer portal provided by Stripe. Please navigate to [Fermyon Cloud Dashboard](https://cloud.fermyon.com) _—> Account settings —> Billing_.
+
+**Can I see multiple past bills?**
+
+Yes. You can view all past bills/invoices for your account in 
+[Fermyon Cloud Dashboard](https://cloud.fermyon.com) under _Account settings —> Billing_.
+
+### Refund
+
+**How do I dispute a charge from Fermyon Cloud?**
+
+We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com)
+
+### Support
+
+**How do I report a problem with Fermyon Cloud?**
+We hope you do not have a problem, but if you do as a Growth or Enterprise Plan user, please contact [support@fermyon.com](mailto:support@fermyon.com). For our Starter users, we recommend reaching out to our [Discord community](https://discord.gg/AAFNfS7NGf) for assistance. 
+
+**What support is offered with Gthe Growth Plan?**
+
+We offer support with a one business day response time via email and a best-effort response time via online chat Monday through Friday. In addition, we offer a vibrant [Discord community](https://discord.gg/AAFNfS7NGf) where common technical questions are often answered faster than one business day, including occasional weekend answers.
+
+**Can I purchase a better level of support?**
+
+Call us and we’ll work to understand the situation and see if we can create a solution to meet your needs.
+
+
+**Does Fermyon Cloud Starter Plan or Growth Plan come with any Service Level Agreements and if so, what are they?**
+
+Fermyon Cloud is an Open Beta service, which currently does not guarentee any Service Level Agreement. 
+
+**Does my use of Fermyon Cloud place any restrictions on the currently open-source Spin?**
+
+No. Spin remains 100% open source and we remain highly committed to both the Spin open source community and open source in general. You can read more about our Open Source Promise [here](https://www.fermyon.com/open-source-promise).
+
+**Do you use data subprocessors? If so, where can I see a list of those you rely on.**
+
+For an up-to-date list of Fermyon’s data subprocessors please visit https://www.fermyon.com/sub-processors.
```

Comment:
```suggestion
For an up-to-date list of Fermyon’s data subprocessors please visit <https://www.fermyon.com/sub-processors>.
```

This ensures the URL is converted into a hyperlink.

https://daringfireball.net/projects/markdown/syntax#autolink